app/assets/scripts/vendor/map.js

```diff
@@ -140,29 +127,31 @@
                   markersToShow.push(markers[i]);
               }
           }
-          var table = $("<table border=1></table>");
+          var table = $("<table border=0></table>");
           $.each(markersToShow, function(i, markerToShow) {
             tableCell = document.createElement('td');
             $(tableCell).attr({
               'style': 'overflow:hidden;width:100%;height:100%;position:absolute'
             });
             tableCellBoldTitle = document.createElement('strong');
             tableCellNewline = document.createElement('br');
```

Comment:
I don't think there is jQuery equivalent of createElement (StackOverflow suggests hard-coding HTML). Looks fine to me.